scripts/build.sh

```diff
@@ -546,27 +546,27 @@ case "$product-$platform-$method" in
       build
     ;;
 
-  Performance-*-xcodebuild)
+  Performance-*-unit)
     # Run unit tests on prod environment with unswizzle capabilities.
     export FPR_UNSWIZZLE_AVAILABLE="1"
     export FPR_AUTOPUSH_ENV="0"
-    pod_gen FirebasePerformance.podspec --platforms=ios --clean
+    pod_gen FirebasePerformance.podspec --platforms="${gen_platform}"
```

Comment:
Do we need to keep the `--clean` option?

Reply:
Not when there are no previous pod_gen's in the job